--- taskjuggler-2.2.0/docs/en/PropertyReference.xml.orig	2005-12-04 16:07:46.000000000 +0100
+++ taskjuggler-2.2.0/docs/en/PropertyReference.xml	2006-03-22 13:06:14.505729750 +0100
@@ -2,7 +2,7 @@
 <?xml-stylesheet type="text/xsl" href="PropertyReference.xsl"?>
 <!DOCTYPE properties [
 <!--  <!ENTITY % DocBookDTD PUBLIC "-//OASIS//DTD DocBook V4.2//EN"
-"/usr/share/sgml/docbook/dtd/4.2xml/docbookx.dtd">
+"/usr/share/sgml/docbook/dtd-4.2-xml/docbookx.dtd">
  %DocBookDTD; -->
  <!ELEMENT properties (property*)>
 
--- taskjuggler-2.2.0/docs/en/taskjuggler.xml.in.orig	2005-12-04 16:07:46.000000000 +0100
+++ taskjuggler-2.2.0/docs/en/taskjuggler.xml.in	2006-03-22 13:01:15.236717250 +0100
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook V4.3//EN" "/usr/share/xml/docbook/schema/dtd/4.3/docbookx.dtd" [
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook V4.3//EN" "/usr/share/sgml/docbook/xml-dtd-4.3/docbookx.dtd" [
  <!ENTITY ChangeLog SYSTEM "ChangeLog.xml">
  <!ENTITY FirstProject SYSTEM "FirstProject.xml">
  <!ENTITY UsageGuide SYSTEM "UsageGuide.xml">
